### PR TITLE
Saving flights

### PR DESCRIPTION
**Pull request by hauglum**
image

### CODE REVIEW

File: src/main/java/no/hauglum/flightlog/service/Scraper.java [hunk 1]
```diff
@@ -56,44 +73,54 @@ public void scrapeTakeOff(int startYear, String takeOffId) {
...
     }
 
+    /**
+     * {@link DocumentFactory} produce documents based on the html-pages found in the Flightlog online.
+     * These docs will be read here and enteties will be extracted.
+     * @param documents
+     */
     private void readDocuments(List<DocumentWrapper> documents) {
-        HashMap<String, Pilot> pilots = new HashMap<String, Pilot>();
+       // HashMap<String, Pilot> pilots = new HashMap<String, Pilot>();
```
Comment (sindrehauglum) [inline src/main/java/no/hauglum/flightlog/service/Scraper.java]: ugly comment

File: src/main/java/no/hauglum/flightlog/service/Scraper.java [hunk 2]
```diff
@@ -56,44 +73,54 @@ public void scrapeTakeOff(int startYear, String takeOffId) {
...
                 if (isADayRow(row)) {
                     flightDay = new FlightDay(row.text());
                     days.add(flightDay);
                 } else if (isAFlightRow(row)) {
                     Elements cells = row.select("td");
 
                     Pilot pilot = parsePilot(cells);
-                    pilots.put(pilot.getFlightlogId(), pilot);
-                    mPilotService.updateOrCreate(pilot);
+        //            pilots.put(pilot.getFlightlogId(), pilot);
```
Comment (sindrehauglum) [inline src/main/java/no/hauglum/flightlog/service/Scraper.java]: ugly comment

File: src/main/java/no/hauglum/flightlog/service/Scraper.java [hunk 3]
```diff
@@ -56,44 +73,54 @@ public void scrapeTakeOff(int startYear, String takeOffId) {
...
+                    flightGroups.add(flightGroup);
+                    mFlightGroupService.updateOrCreate(flightGroup);
 
                 } else {
                     mLogger.debug("some other row in table found");
                 }
             }
         }
+
+        //mFlightGroupService.saveAll(flightGroups);
```
Comment (sindrehauglum) [inline src/main/java/no/hauglum/flightlog/service/Scraper.java]: ugly comment

File: src/main/java/no/hauglum/flightlog/service/Scraper.java [hunk 4]
```diff
@@ -56,44 +73,54 @@ public void scrapeTakeOff(int startYear, String takeOffId) {
...
         mLogger.info("Sluttrapport" );
         mLogger.info("Antall flydager: " + days.size());
-        mLogger.info("Antall unike piloter: " + pilots.size());
+        //mLogger.info("Antall unike piloter: " + pilots.size());
```
Comment (sindrehauglum) [inline src/main/java/no/hauglum/flightlog/service/Scraper.java]: ugly comment

**Review by sindrehauglum — CHANGES_REQUESTED** [review]
ugly comment

File: src/main/java/no/hauglum/flightlog/service/Scraper.java (hunk 1 above)
Comment (hauglum) [reply src/main/java/no/hauglum/flightlog/service/Scraper.java]: fixed

File: src/main/java/no/hauglum/flightlog/service/Scraper.java (hunk 2 above)
Comment (hauglum) [reply src/main/java/no/hauglum/flightlog/service/Scraper.java]: fixed

File: src/main/java/no/hauglum/flightlog/service/Scraper.java (hunk 3 above)
Comment (hauglum) [reply src/main/java/no/hauglum/flightlog/service/Scraper.java]: fixed

**Comment by hauglum** [comment]
fixed

**Review by sindrehauglum — APPROVED** [review]

**Pull request merged**